components/brave_shields/core/browser/DEPS

```diff
@@ -0,0 +1,2 @@
+include_rules = [
```

Comment:
why is this one empty?

Reply:
Honestly I added this thinking I'll add something to it. But never needed to. I just removed it.